bcda/web/api.go

```diff
@@ -501,6 +501,30 @@ func healthCheck(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+/*
+	swagger:route GET /api/v1/welcome metadata welcome
```

Comment:
Should this be included in the `auth` section instead of `metadata` section on Swagger?